modules/project_cleanup/function_source/main.go

```diff
@@ -173,6 +193,36 @@ func getLabelsMapFromEnv(envVariableName string) map[string]string {
 	return labels
 }
 
+func getTagKeysListFromEnv(envVariableName string) []string {
+	targetExcludedTagKeys := os.Getenv(envVariableName)
+	logger.Println("Try to get Tag Keys list")
+	if targetExcludedTagKeys == "" {
+		logger.Printf("No Tag Keys provided.")
+		return nil
+	}
+
+	var tagKeys []string
+	err := json.Unmarshal([]byte(targetExcludedTagKeys), &tagKeys)
+	if err != nil {
+		logger.Printf("Fail to get Tag Keys list from [%s] env variable, error [%s]", envVariableName, err.Error())
+	} else {
+		logger.Printf("Got Tag Keys list [%s] from [%s] env variable", tagKeys, envVariableName)
+	}
+	return tagKeys
+}
+
+func getCleanUpTagKeysOrTerminateExecution() bool {
+	cleanUpTagKeys, exists := os.LookupEnv(CleanUpTagKeys)
+	if !exists {
+		logger.Fatalf("Clean up Tag Keys flag not set [%s]. Specify correct value, Please.", CleanUpTagKeys)
```

Comment:
nit: Our typical error message style doesn't include "please".  Perhaps something instruction such as "Set the [%s] environment variable and try again."

If you really want to keep "please", I'd recommend something like: "Please set the required [%s] environment variable".

Reply:
@apeabody we have another 2 cases of previously existing messages with "Please".
Should we update them all for consistency?

Reply:
Hi @daniel-cit - I would suggest doing so.

Reply:
fixed